src/main/java/com/redhat/red/build/finder/Main.java

```diff
@@ -114,6 +115,7 @@ public static void main(String[] args) {
         options.addOption(Option.builder("k").longOpt("checksum-only").numberOfArgs(0).required(false).desc("Only checksum files and do not find builds. Default: " + ConfigDefaults.CHECKSUM_ONLY + ".").build());
         options.addOption(Option.builder("t").longOpt("checksum-type").argName("type").numberOfArgs(1).required(false).type(String.class).desc("Set checksum type (" + Arrays.stream(KojiChecksumType.values()).map(KojiChecksumType::name).collect(Collectors.joining(", ")) + "). Default: " + ConfigDefaults.CHECKSUM_TYPE + ".").build());
         options.addOption(Option.builder("a").longOpt("archive-type").argName("type").numberOfArgs(1).required(false).desc("Add a Koji archive type to check. Default: [" + ConfigDefaults.ARCHIVE_TYPES.stream().collect(Collectors.joining(", ")) + "].").type(List.class).build());
+        options.addOption(Option.builder("e").longOpt("archive-extension").argName("extension").numberOfArgs(1).required(false).desc("Add a Koji archive type extension to check. Default: [" + ConfigDefaults.ARCHIVE_EXTENSIONS.stream().collect(Collectors.joining(", ")) + "].").type(List.class).build());
         options.addOption(Option.builder("x").longOpt("exclude").numberOfArgs(1).argName("pattern").required(false).desc("Add a pattern to exclude from build lookup. Default: [" + ConfigDefaults.EXCLUDES.stream().collect(Collectors.joining(", ")) + "].").build());
         options.addOption(Option.builder().longOpt("koji-hub-url").numberOfArgs(1).argName("url").required(false).desc("Set Koji hub URL.").build());
```

Comment:
Do we need excludes now we have includes via the known archive types?

Reply:
> I also suspect the configuration file in my $HOME is interfering with the tests - maybe they should all start with their own config file?

There was a bug where the local configuration was overwritten by the current configuration. This also caused a `NullPointerException` depending on the test order because the configuration didn't necessarily match the default arguments. Please verify that your local configuration is correct, particular the checksum options. The latest commit should fix the overwrite issue.

Reply:
> Do we need excludes now we have includes via the known archive types?

Yes. For example, if it has type or extension xml, but we don't want to analyze all xml files, just pom.xml, then we need the ability to exclude, because the extension itself globs too much.